web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
```

Comment:
I agree with @JoshSmith that adding a `stripe_id` column is the way to go. If we replace `id` with a value that's tied to a third party value, we're essentially making `id` a foreign key, which could be confusing in the future.